pycomfoconnect/comfoconnect.py

```diff
@@ -281,16 +329,21 @@ def _get_reply(self, confirm_type=None, timeout=5, use_queue=True):
                     self._queue.put(message)
 
             if time.time() - start > timeout:
-                raise ValueError('Timeout waiting for response.')
+                # why raise?
```

Comment:
The raise is because we are expecting a message, and it didn't arrive. Something went wrong, so that's why I raised an exception.